database/pgsql/migrations/00020_languagelayer_lineage_pk.go

```diff
@@ -0,0 +1,14 @@
+package migrations
+
+import "github.com/remind101/migrate"
+
+func init() {
+	RegisterMigration(migrate.Migration{
+		ID: 20,
+		Up: migrate.Queries([]string{
+			`ALTER TABLE LanguageLayer DROP CONSTRAINT IF EXISTS languagelayer_pkey`,
+			`ALTER TABLE LanguageLayer DROP CONSTRAINT IF EXISTS languagelayer_layer_name_key`,
```

Comment:
so dropping this constraint removes the `UNIQUE` part of `layer_name VARCHAR(128) NOT NULL UNIQUE`?